### DISCUSSION TITLE
Trigger github actions manually

### DISCUSSION THREAD

**Discussion by SaeedHK** [community/community#4578] [Actions] [answered]
To control better GitHub actions, some projects may require that you only trigger actions for specific commits. I was wondering if it would make sense to add an event, like "manual", to GitHub actions to only get triggered when we select it manually, like hitting a button next to each commit.

**Comment by ChildishGiant** [comment, marked as answer]
This was added just over a year ago :wink: 
https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

**Reply by SaeedHK** [reply]
It's cool, thank you very much for sharing!
